### PR TITLE
Update the kubelet log pod status to level 6 as it is so big

### PR DESCRIPTION
**Pull request by pacoxu**
#### What type of PR is this?
/kind bug

#### Which issue(s) this PR fixes:
xref: https://github.com/kubernetes/kubernetes/issues/99803#issuecomment-806501942

example to show the diff between 1.20 and 1.21
https://github.com/kubernetes/kubernetes/issues/99803#issuecomment-805624215

#### Special notes for your reviewer:
/cc serathius ehashman 

#### Does this PR introduce a user-facing change?
```release-note
None
```

### CODE REVIEW

File: pkg/kubelet/pleg/generic.go [hunk 1]
```diff
@@ -383,7 +383,10 @@ func (g *GenericPLEG) updateCache(pod *kubecontainer.Pod, pid types.UID) error {
 	// GetPodStatus(pod *kubecontainer.Pod) so that Docker can avoid listing
 	// all containers again.
 	status, err := g.runtime.GetPodStatus(pod.ID, pod.Name, pod.Namespace)
-	klog.V(4).ErrorS(err, "PLEG: Write status", "pod", klog.KRef(pod.Namespace, pod.Name), "podStatus", status)
+	klog.V(4).ErrorS(err, "PLEG: Write status", "pod", klog.KRef(pod.Namespace, pod.Name))
+	if klog.V(6).Enabled() {
+		klog.ErrorS(err, "PLEG: Write status", "pod", klog.KRef(pod.Namespace, pod.Name), "podStatus", status)
+	}
```
Comment (ehashman) [inline pkg/kubelet/pleg/generic.go:389]: As written, this double-logs, but also seems kinda redundant... would we not write this as

```suggestion
	if klog.V(6).Enabled() {
		klog.V(6).ErrorS(err, "PLEG: Write status", "pod", klog.KRef(pod.Namespace, pod.Name), "podStatus", status)
	} else {
		klog.V(4).ErrorS(err, "PLEG: Write status", "pod", klog.KRef(pod.Namespace, pod.Name))
	}
```

or

```suggestion
	klog.V(4).ErrorS(err, "PLEG: Write status", "pod", klog.KRef(pod.Namespace, pod.Name))
	if klog.V(6).Enabled() {
		klog.V(6).ErrorS(err, "PLEG: Write status", "pod", klog.KRef(pod.Namespace, pod.Name), "podStatus", status)
	}
```

**Review by ehashman — COMMENTED** [review]
/priority critical-urgent
/triage accepted
/kind regression

/cc @dims @serathius 
for review

/cc @palnabarun 
for RT acceptance re: regression

**Comment by palnabarun** [comment]
/milestone v1.21

as it fixes a regression.

File: pkg/kubelet/pleg/generic.go (hunk 1 above)
Comment (pacoxu) [reply pkg/kubelet/pleg/generic.go]: I prefer the first one.

**Comment by MonzElmasry** [comment]
@ehashman @palnabarun @dims @serathius mind if someone takes a look at these changes, CI is successful.

**Review by ehashman — APPROVED** [review]
/lgtm

**Pull request merged**